docs/src/main/sphinx/release/release-407.md

```diff
@@ -0,0 +1,104 @@
+# Release 407 (13 Feb 2023)
+
+## General
+
+* Add support for adaptive task request sizing. It can be enabled with the
+  `query.remote-task.enable-adaptive-request-size` configuration property and
+  configured with the `query.remote-task.max-request-size`,
+  `query.remote-task.request-size-headroom`, and
+  `query.remote-task.guaranteed-splits-per-task` configuration properties or
+  their respective session properties. ({issue}`15721`)
+* Add support for correlated queries with single-row values. ({issue}`15989`)
+* Add support for configuring the minimum number of rows and
```

Comment:
Has the default changed? What's the user visible impact? If this is not visible, remove the entry. 

Also, why are these "optimizer.xxx" options? The seem to be related to the runtime, in which case they should be "query.xxxx". cc @gaurav8297 @sopel39